src/AksArc/AksArc.Autorest/custom/Get-AzAksArcCluster.ps1

```diff
@@ -0,0 +1,129 @@
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code
+# is regenerated.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Gets the provisioned cluster instance
+.Description
+Gets the provisioned cluster instance
+.Example
+Get-AzAksArcCluster -ClusterName azps_test_cluster -ResourceGroup azps_test_group
+
+.Outputs
+Microsoft.Azure.PowerShell.Cmdlets.AksArc.Models.IProvisionedCluster
+.Link
+https://learn.microsoft.com/powershell/module/az.aksarc/get-azaksarccluster
+#>
+function Get-AzAksArcCluster {
+[OutputType([Microsoft.Azure.PowerShell.Cmdlets.AksArc.Models.IProvisionedCluster])]
+[CmdletBinding(DefaultParameterSetName='Get', PositionalBinding=$false)]
+param(
+    [Parameter(Mandatory)]
+    [Alias('Name')]
+    [Microsoft.Azure.PowerShell.Cmdlets.AksArc.Category('Path')]
+    [System.String]
+    # The name of the Kubernetes cluster on which get is called.
+    ${ClusterName},
+
+    [Parameter(Mandatory)]
+    [Alias('resource-group')]
```

Comment:
Why would you have the Alias which do not match Azure PowerShell naming conventions?

Reply:
Thanks resolved. Just wanted to make it more similar to our cli extension but it isn't necessary.